src/test/resources/db/insert-movies.sql

```diff
@@ -0,0 +1,265 @@
+--
```

Comment:
shouldn't be presented in current project

Reply:
removed